package.json

```diff
@@ -72,7 +72,7 @@
     "gulp-replace": "^0.5.3",
     "gulp-sass": "^2.0.0",
     "gulp-sourcemaps": "^1.2.8",
-    "gulp-stylus": "^2.0.0",
+    "gulp-stylus": ">=2.1.2",
```

Comment:
Please same format as other dependencies: `^2.1.2`. Your `>=` is too vague and can breake anytime.